tutorials/text/text_props.py

```diff
@@ -149,9 +149,9 @@
 # +---------------------+----------------------------------------------------+
 # | rcParam             | usage                                              |
 # +=====================+====================================================+
-# | ``'font.family'``   | List of either names of font or ``{'cursive',      |
-# |                     | 'fantasy', 'monospace', 'sans', 'sans serif',      |
-# |                     | 'sans-serif', 'serif'}``.                          |
+# | ``'font.family'``   | List of names of user-defined font families and/or |
```

Comment:
All font families which not denoted by this set: `{'cursive', 'serif', 'sans-serif', 'fantasy', 'monospace'} are user-defined font families.

We basically draw a line between "generic-families" vs "family-names", see [this link](https://drafts.csswg.org/css-fonts-3/#font-family-prop):

> There are two types of font family names:
    family-name
    The name of a font family of choice such as Helvetica or Verdana in the previous example.
    generic-family
    The following generic family keywords are defined: ‘serif’, ‘sans-serif’, ‘cursive’, ‘fantasy’, and ‘monospace’.

Reply:
Though I agree, we could use a better name than 'user-defined' font families, because it might give the impression that a user himself has to define that font.

Any suggestions?

Reply:
```suggestion
# | ``'font.family'``   | List of font families (installed on the users machine) and/or |
```

Reply:
Suggest before this table something like:

```
Matplotlib can use font families installed on the user's computer, i.e. Helvetica, Times, etc, as well as other user-installed fonts.  Font families can also be specified with generic names like 'serif', 'sans-serif', in which case a fallback list is used:
```

Reply:
So maybe here, link the rcParam so folks can see the default list?